services/install-domoticz.sh

```diff
@@ -0,0 +1,240 @@
+#!/bin/bash
+
+function install {
+  # create service directory
+  mkdir -p /srv/domoticz
+
+  # create yml(s)
+  {
+    echo "services:"
+    echo "  domoticz:"
+    echo "    image: linuxserver/domoticz"
+    echo "    ports:"
+    echo "      - \"6144:6144\""
+    echo "    volumes:"
+    echo "      - \"/srv/domoticz:/root/.domiticz\""
+    echo "version: \"2\""
+  } > /srv/domoticz/domoticz.yml
+
+  # add autorun
+  {
+    echo "domoticz_autorun=true"
+    echo
+    echo "if [ \"\$domoticz_autorun\" = true ]; then"
+    echo "  treehouses services domiticz up"
+    echo "fi"
+    echo
+    echo
+  } > /srv/domoticz/autorun
+}
+
+# environment var
+function uses_env {
+  echo false
+}
+
+# add supported arm(s)
+function supported_arms {
+  echo "v7l"
+}
+
+# add port(s)
+function get_ports {
+  echo "6144"
+}
+
+# add size (in MB)
+function get_size {
+  echo "650"
+}
+
+# add info
+function get_info {
+  echo "https://github.com/domoticz/domoticz"
+  echo
+  echo "\"Domoticz is a Home automation system that lets you monitor and configure various devices.\""
+}
+
+# add svg icon
+function get_icon {
+  cat <<EOF
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 234.6 193.6">
```

Comment:
Could you condense this long svg link using https://jakearchibald.github.io/svgomg/?